src/components/Facility/FacilityUsers.tsx

```diff
@@ -48,13 +61,15 @@ export default function FacilityUsers(props: { facilityId: number }) {
 
       <UserListView
         users={userListData?.results ?? []}
-        onSearch={(username) => updateQuery({ username })}
-        searchValue={qParams.username}
+        onSearch={(username) => handleSearch("username", username)}
```

Comment:
No need to complicate this:

```suggestion
       onSearch={(username) => updateQuery({ username })}
```

What we had earlier is enough.